src/megaapi.cpp

```diff
@@ -3300,12 +3300,12 @@ void MegaApi::syncFolder(const char *localFolder, MegaNode *megaFolder, MegaRequ
 
 void MegaApi::syncFolder(const char *localFolder, const char *name, MegaHandle megaHandle, MegaRequestListener *listener)
 {
-    pImpl->syncFolder(localFolder, name, megaHandle, NULL, listener);
+    pImpl->syncFolder(localFolder, name, megaHandle, SyncConfig::Type::TYPE_TWOWAY, NULL, listener);
```

Comment:
I can do without mentioning `Type::` scope. Not sure which is more or less readable though.